landlab/components/soil_moisture/funcs_for_including_runon_in_soil_moisture.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jan 17 11:48:13 2017
+
+@author: Sai Nudurupati & Erkan Istanbulluoglu
+"""
+
+import numpy as np
+from landlab.components import FlowRouter  # , DepressionFinderAndRouter
+
+
+def get_ordered_cells_for_soil_moisture(grid, outlet_id=None):
+    """
+    Runs Landlab's FlowRouter and DepressionFinderAndRouter to
+    route flow. Also orders the cells in the descending order of
+    channel length (upstream cell order).
+
+    Parameters:
+    ==========
```

Comment:
Use *-* for underline, not *=*. Remove the colon.

Reply:
done